.circleci/config.yml

```diff
@@ -66,6 +66,37 @@ jobs:
           name: Check for Updated Package name
           command: node -e "if (require('./package.json').name=='@adobe/helix-library') { process.exit(1) };"
 
+      - run:
+          name: Semantic Orb Release
+          command: |
+              # analyze commits to find out if any orbs need to be updated
+              npm run semantic-release --dry-run --analyze-commits | tee /tmp/commit-analysis.log
+              segment=$(cat /tmp/commit-analysis.log | sed '/^.* complete: */!d; s///;q' | sed 's/^\(.*\) release/\1/')
+              if [[ "${segment}" =~ ^(major|minor|patch)$ ]]; then
+                echo "Commit analysis suggests ${segment} release, identifying affected orbs"
+                for path in .circleci/orbs/*
+                do
+                  if [[ "${path}" =~ .*/README.md$ ]]; then
+                    continue
+                  fi
```

Comment:
One line `if` statements can be replaced with `condition && then-statement`, e.g.:
```
[[ "${path}" =~ .*/README.md$ ]] && continue
```